method/BANKSY/banksy.r

```diff
@@ -168,8 +168,14 @@ k_geom <- config$k_geom
 npcs <- config$npcs
 resolution <- config$resolution
 method <- config$method
-assay_name <- "logcounts"
+assay_name <- "normcounts"
 set.seed(seed)
+
+# Normalization to mean library size
+spe <- computeLibraryFactors(spe)
+assay(spe, aname) <- normalizeCounts(spe, log = FALSE)
```

Comment:
`aname` is not defined elsewhere? Should this be `assay_name`?

Reply:
Good catch!